src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -88,4 +89,16 @@ object EventMessages {
       }
     }
   }
+
+  implicit val exportStatusEventMessages: Messages[ExportStatusEvent] = new Messages[ExportStatusEvent] {
+    override def email(incomingEvent: ExportStatusEvent): Option[Email] = {
+      println(s"Skipping email for export complete message for consignment ${incomingEvent.consignmentId}")
+      Option.empty
+    }
+
+    override def slack(incomingEvent: ExportStatusEvent): Option[String] = {
+      val message = s"The export for the consignment ${incomingEvent.consignmentId} has ${if (incomingEvent.success) "completed" else "failed"}"
+      SlackMessage(List(SlackBlock("section", SlackText("mrkdwn", message)))).asJson.noSpaces.some
```

Comment:
Given that all the Slack message methods have to include the `.asJson.noSpaces`, maybe it would be better for this method to returns a SlackMessage rather than a String and commonise the serialization?

Reply:
Good point, thanks. I've changed it to return a SlackMessage and called `.asJson.noSpaces` in the `sendSlackMessage` method.